examples/plot_interpolate_bad_channels.py

```diff
@@ -1,10 +1,14 @@
 """
-================================================
-Interpolate bad channels using spherical splines
-================================================
+=============================================
+Interpolate bad channels for MEG/EEG channels
+=============================================
 
-This example shows how to interpolate bad EEG channels using spherical
-splines as described in [1].
+This example shows how to interpolate bad MEG/EEG channels
+    - Using spherical splines as described in [1] for EEG data.
```

Comment:
I'm pretty sure for this to render properly you need an empty newline before the list starts